src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -390,9 +405,7 @@ def get_eligible_locations_principals(self) -> dict[str, dict]:
             logger.error(msg)
             raise ResourceDoesNotExist(msg) from None
 
-        permission_mappings = self._installation.load(
-            list[AWSRoleAction], filename=AWSResourcePermissions.INSTANCE_PROFILES_FILE_NAMES
```

Comment:
Why did you remove this? Is this file managed by AwsACL? What if someone renames the filename in resource permissions class?

Revert this removal.

Reply:
I am calling PrincipalACL from AWSResourcePermission and this code is calling AWSResourcePermission and I am getting circular references

Reply:
Good point

Reply:
https://github.com/databrickslabs/ucx/issues/1572 created issue to track this